src/WCSimConstructPMT.cc

```diff
@@ -138,12 +138,12 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
     // gel layer ("container") and vessel.
     solidWCPMT =
       new G4Sphere("WCPMT",
-		   vessel_radius_curv - mPMT_outer_material_d - pmtModuleHeight, //rMin, 54mm is position of support structure
-		   vessel_radius_curv - mPMT_outer_material_d,          //rMax
+		   vessel_radius_curv - mPMT_outer_material_d - pmtModuleHeight, //rMin = 342 - 10 - 54 = 278 mm, 54mm is position of support structure
```

Comment:
In the above, '''pmtModuleHeight''' is declared with 59.62*CLHEP::mm. So the comment, 342 - 10 - 54 = 278mm, seems to be strange.

Reply:
Yes, the comment needs to be changed

Reply:
@nickwp could you quickly fix this?